source/manual/data-gov-uk-monitoring.html.md

```diff
@@ -23,11 +24,11 @@ Container metrics are scraped using the [PaaS Metric Exporter][paas-metric-expor
 
 ## Pingdom
 
-Pingdom checks whether `data.gov.uk` exists and emails `data.gov.uk@digital.cabinet-office.gov.uk` with updates. Maintenace of this service forms part of `#govuk-2nd-line`.
+Pingdom monitors `https://data.gov.uk` uptime and alerts [PagerDuty] when downtime is detected. Maintenace of this service forms part of `#govuk-2nd-line`.
 
 ## Sentry
 
-[Sentry] receives application errors and emails `data.gov.uk@digital.cabinet-office.gov.uk` with updates. The Sentry pages for each app can be found on the [Find] and [Publish] app pages.
+[Sentry] monitors application errors. The Sentry pages for each app can be found on the [Find] and [Publish] app pages.
```

Comment:
So where's it going now?

Reply:
Email notifications are currently sent to a number of named individuals. The exceptions are viewable be all who have access to the GOV.UK Sentry account. Platform Health are going to re-organise the user grouping at the start of next mission.